docs/usage.md

```diff
@@ -69,119 +115,360 @@ cat ./compilation.json
   and in most cases, _System Integrity Protection_ needs to be turned off.
   See the [README](/README.md#mac-os-x) for details.
 
-## Step 2: Analyze your code <a name="step-2"></a>
-Once the build is logged successfully (and the `compilation.json`) was created,
-you can analyze your project.
+## Step 2: Analyze your code<a name="step-2"></a>
+Once the build is logged successfully and the `compile_commands.json` was
+created, you can analyze your project.
 
-1. Run the analysis:
+### Run the analysis<a name="run-the-analysis"></a>
 ```sh
-CodeChecker analyze compilation.json -o ./reports
+CodeChecker analyze ./compile_commands.json --enable-all --output ./reports
 ```
-2. View the analysis results in the command line
+
+However the compilation of the project (here the example program) is performed
+by `gcc`, the CodeChecker uses `clang` to analyze sources of the project.
+During analysis, CodeChecker compiles/analyses sources again. The analysis
+process generally uses more time. It would be good to mention here that if you
+want to speed up analysis specify a higher value for the `--jobs` option.
+
+In the above command the `--enable-all` means that a subset of checker
+are run. `enable-all` chooses a predefined "group" of checkers. The further
+info on available checkers use
+
+```sh
+# List available checkers.
+CodeChecker checkers --help
+# Show more information about the checkers.
+CodeChecker checkers --details --help
+# List profiles.
+CodeChecker checkers --profile list --details
+# List checkers which are in the sensitive profile.
+CodeChecker checkers --profile sensitive --details
+```
+commands.
+
+The `./reports` directory is the "database" of CodeChecker that allows to
+manage further working steps.
+
+### View the analysis results in the command line<a name="view-analysis-results"></a>
 ```sh
 CodeChecker parse ./reports
 ```
 
 Hint:
- You can do the 1st and the 2nd step in one round by executing `check`
+You can do the 1st and the 2nd step in one round by executing `check`
 ```sh
-cd tmux
+cd <repo root dir>/docs/examples
 make clean
-CodeChecker check -b "make" -o ./reports
+rm -rf ./reports
+CodeChecker check --build "make" --output ./reports --enable-all
```

Comment:
Use the `--clean` option to clear the report directory.
```suggestion
rm -rf ./reports
CodeChecker check --build "make" --output ./reports --clean
```

Also don't use the --enable option.

Reply:
Done.